lab-beeksma/lib/dataFromFile.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+
+function logSpecificFile(filename, cb) {
+  fs.readFile(`${__dirname}/../assets/${filename}`, function (err, res) {
+    if(err){
+      cb(err,undefined);
+    }
+    cb(err,res.slice(0,8).toString('hex'));
```

Comment:
Typically `err` here would be replaced by `null` explicitly, even though at this point `err` is known to be falsey.